### ISSUE TITLE
'NullIO' object has no attribute 'write_line'

### ISSUE THREAD

**Issue by darrenleeweber** [sdispater/poetry#1060]
- [x] I am on the [latest](https://github.com/sdispater/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] If an exception occurs when executing a command, I executed it again in debug mode 

```
$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04.2 LTS"

$ poetry --version
Poetry 0.12.14
```

Sorry, can't provide proprietary `pyproject.toml` but it mostly has vanilla pypi packages and this bug doesn't look like a package problem.

Reverting to `Poetry 0.12.11` doesn't have this bug.

## Issue

```
$ poetry -v install

... snipped .... installs all the packages OK ...

[AttributeError]                  
'NullIO' object has no attribute 'write_line'  
                                                 
Exception trace:
 /home/joe/.poetry/lib/poetry/_vendor/py3.7/cleo/application.py in run() at line 94
   status_code = self.do_run(input_, output_)
 /home/joe/.poetry/lib/poetry/console/application.py in do_run() at line 88
   return super(Application, self).do_run(i, o)
 /home/joe/.poetry/lib/poetry/_vendor/py3.7/cleo/application.py in do_run() at line 197
   status_code = command.run(input_, output_)
 /home/joe/.poetry/lib/poetry/console/commands/command.py in run() at line 77
   return super(BaseCommand, self).run(i, o)
 /home/joe/.poetry/lib/poetry/_vendor/py3.7/cleo/commands/base_command.py in run() at line 146
   status_code = self.execute(input_, output_)
 /home/joe/.poetry/lib/poetry/_vendor/py3.7/cleo/commands/command.py in execute() at line 107
   return self.handle()
 /home/joe/.poetry/lib/poetry/console/commands/install.py in handle() at line 77
   builder.build()
 /home/joe/.poetry/lib/poetry/masonry/builders/editable.py in build() at line 17
   return self._setup_build()
 /home/joe/.poetry/lib/poetry/masonry/builders/editable.py in _setup_build() at line 25
   self._io.write_line(
```

**Comment by chrisbennight** [comment]
Same issue - I think this:
https://github.com/sdispater/poetry/blob/dad6cda25250d061f94325a10dd074453eee6e50/poetry/masonry/builders/editable.py#L25

might be intended to be `self._io.writeln(` instead (based on the methods on NullIO

**Comment by Djailla** [comment]
Same issue here :(

**Comment by seyhbold** [comment]
Had the same issue and upgraded to 0.12.15. It is working now for me.

**Comment by sdispater** [comment]
As @seyhbold pointed out, it has been fixed the the `0.12.15` release.
